tests/test_management.py

```diff
@@ -22,7 +22,7 @@
 from django.core.management.base import CommandError
 from django.utils import timezone
 
-from enterprise import lms_api
+from enterprise.api_client import lms
```

Comment:
This import can be:
```
from enterprise.api_client import lms as lms_api
```
Then you can revert the rest of the changes in this file.